jose.py

```diff
@@ -209,13 +209,14 @@ def decrypt(jwe, jwk, adata='', validate_claims=True, expiry_seconds=None):
     # decrypt body
     ((_, decipher), _), ((hash_fn, _), mod) = JWA[header['enc']]
 
-    if header.get(_TEMP_VER_KEY) == _TEMP_VER:
+    version = header.get(_TEMP_VER_KEY, None)
+    if version:
```

Comment:
`version` cannot be `0`. Versioning was introduced to allow working with old tokens while fixing some bugs in the library making it incompatible with the standard. This is another bug like that, meaning even version 1 tokens do not comply with the standard. When versioning was introduced, version `1` was the first one to be used, now replaced with version 2.

If a token with version `0` is found, it was definitely not defined with this library. Therefore, there's no *right* way to parse it, and it doesn't make sense bothering about it.